Storage/REST-API.md

```diff
@@ -0,0 +1,183 @@
+# RESTful binding for Storage, RFC
+
+This document describes the RESTful API with CRUD functionality for Current Storage.
+
+### Bindings semantics
+`TODO: Describe storage field <-> endpoint bindings.`
+
+### Response schema
+
+The response body is always either a valid JSON or empty.
```

Comment:
The empty body should be declared a (very limited) special case. Both successful and unsuccessful responses should always render a JSON.